commit 421a85119fd80892982c8c053c4c23c5aef55042 (refs/changes/60/4260/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-06-19T12:26:34-07:00 (1 year, 4 months ago)
    
    TRITON-525 vmapi docs incorrectly state "dns_domain" affects /etc/hosts

diff --git a/docs/index.md b/docs/index.md
index c70154b..cfb24e2 100644
--- a/docs/index.md
+++ b/docs/index.md
@@ -52,7 +52,7 @@ update time.
 | destroyed                | Date                          | The time at which the VM was destroyed in ISO 8601 format                                                                                                                                                                 | Yes                 | No     | No     |
 | datasets                 | Array                         | VM datasets                                                                                                                                                                                                               | Yes                 | No     | No     |
 | delegate_dataset         | Boolean                       | Delegate a data dataset to the VM                                                                                                                                                                                         | No                  | Yes    | No     |
-| dns_domain               | String                        | Domain value for /etc/hosts and /etc/resolv.conf (max length 255 chars)                                                                                                                                                   | No                  | Yes    | No     |
+| dns_domain               | String                        | Search domain value for /etc/resolv.conf (max length 255 chars)                                                                                                                                                   | No                  | Yes    | No     |
 | do_not_inventory         | Boolean                       | The primary use-case of this attribute is for test VMs that are created but you don't want their existence propagated up to VMAPI since they'll be short-lived and its lifecycle will be physically managed in the server | No                  | Yes    | Yes    |
 | firewall_enabled         | Boolean                       | Enable firewall for the VM                                                                                                                                                                                                | Yes                 | Yes    | Yes    |
 | fs_allowed               | String (comma-separated list) | Filesystems allowed for the VM                                                                                                                                                                                            | No                  | Yes    | Yes    |
